doc/generate.py

```diff
@@ -3,23 +3,35 @@
 import sys
 import os
 from contextlib import suppress
-
 import toucan_connectors
```

Comment:
You do not need to import it anymore ?

Reply:
I removed this import as it is no longer used.